CommunityToolkit.Mvvm/Properties/AssemblyInfo.cs

```diff
@@ -7,4 +7,4 @@
 // See notes in CommunityToolkit.HighPerformance for more info
 [module: SkipLocalsInit]
 
-[assembly: InternalsVisibleTo("CommunityToolkit.Mvvm.Internals.UnitTests")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("CommunityToolkit.Mvvm.Internals.UnitTests, PublicKey=002400000480000094000000060200000024000052534131000400000100010041753AF735AE6140C9508567666C51C6AB929806ADB0D210694B30AB142A060237BC741F9682E7D8D4310364B4BBA4EE89CC9D3D5CE7E5583587E8EA44DCA09977996582875E71FB54FA7B170798D853D5D8010B07219633BDB761D01AC924DA44576D6180CDCEAE537973982BB461C541541D58417A3794E34F45E6F2D129E2")]
```

Comment:
Here too? These are shipping assemblies!! Do we need this in final binary?

Reply:
It's very normal for these to exist in a shipping binary. Go check almost any major NuGet package.

Reply:
Just because every package out there does it doesn't mean we have too! I was just wondering if it's best practice!

Reply:
It's not like you can drop this attribute anyhow. The whole reason for the PR is to bring signing back up to parity with v7. Read the previous discussion on the PR about `IVT` errors.

Reply:
I actually welcome this change but I didn't want to leave the attribute in the release. I could guard it but CI then would require two builds, one for tests and one for release.

There's no way to do it otherwise, so, I didn't do it in my refactor PR!